ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2085,7 +2085,7 @@ def test_numpy_instance_itruediv__(
     frontend,
 ):
     input_dtypes, xs = dtype_and_x
-
+    assume(not np.any(np.isclose(xs[1], 0.0)))
```

Comment:
this is probably unnecessary, as division by 0 results in `inf` which can be handled, we can remove this line.

Reply:
okay let me get to that